src/typing_extensions.py

```diff
@@ -585,7 +568,7 @@ class _ProtocolMeta(type(typing.Protocol)):
         # but is necessary for several reasons...
         #
         # NOTE: DO NOT call super() in any methods in this class
-        # That would call the methods on typing._ProtocolMeta on Python 3.8-3.11
+        # That would call the methods on typing._ProtocolMeta on Python 3.9-3.11
```

Comment:
this avoids us having to update this again when we drop support for Python 3.9 😆

```suggestion
        # That would call the methods on typing._ProtocolMeta on Python <=3.11
```